File: Android/pool/src/main/java/com/felipearpa/tyche/pool/creator/PoolFromLayoutCreatorRoute.kt
package com.felipearpa.tyche.pool.creator

import kotlinx.serialization.Serializable

@Serializable
data class PoolFromLayoutCreatorRoute(
    val gamblerId: String,
    val preselectedPoolLayoutId: String? = null,
    val preselectedPoolName: String? = null,
)

File: iOS/Pool/Sources/Pool/Creator/PoolFromLayoutCreatorRoute.swift
public struct PoolFromLayoutCreatorRoute : Hashable {
    public let preselectedPoolLayoutId: String?
    public let preselectedPoolName: String?

    public init(
        preselectedPoolLayoutId: String? = nil,
        preselectedPoolName: String? = nil,
    ) {
        self.preselectedPoolLayoutId = preselectedPoolLayoutId
        self.preselectedPoolName = preselectedPoolName
    }
}
